### PR TITLE
[CDAP-17567] Fixes UI to show drafts from user store even if pipeline studio service is down

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA - https://cdap.atlassian.net/browse/CDAP-17567

### CODE REVIEW

**Review by elfenheart — APPROVED** [review]

File: cdap-ui/app/cdap/components/PipelineList/DraftPipelineView/store/ActionCreator.ts [hunk 1]
```diff
@@ -43,16 +45,29 @@ function getOldDrafts(oldDrafts) {
...
   Observable.forkJoin<
-    IDraft[],
+    any,
     any /** setting as 'any'. We should remove user store reference in the next major release */
   >(
     MyPipelineApi.getDrafts({
       context: 'default',
-    }),
+    })
+      .map((res) => res)
```
Comment (njbriggs) [inline cdap-ui/app/cdap/components/PipelineList/DraftPipelineView/store/ActionCreator.ts]: What does this `.map` do?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/PipelineList/DraftPipelineView/store/ActionCreator.ts]: Doesn't seem to require. I was under the assumption that the map is required for catch'ing the errors. Have removed it.

**Pull request merged**